android/app/src/main/java/app/organicmaps/widget/placepage/EditBookmarkFragment.java

```diff
@@ -76,6 +83,20 @@ public static void editBookmark(long categoryId, long bookmarkId, @NonNull Conte
     fragment.show(manager, name);
   }
 
+  public static void editTrack(long categoryId, long trackId, Context context, FragmentManager manager, @Nullable EditBookmarkListener listener)
+  {
+    final Bundle args = new Bundle();
+    args.putInt(EXTRA_BOOKMARK_TYPE, 2);
```

Comment:
Please introduce constants for `EXTRA_BOOKMARK_TYPE` values. Possible values are `1` and `2`. It would be better to name it like `TYPE_BOOKMARK` and `TYPE_TRACK`.

Reply:
This method is used to start the editing fragment